.github/workflows/release.yml

```diff
@@ -28,6 +28,7 @@ jobs:
 
   publish_ublox:
     name: Publish
```

Comment:
Could you change this to `Publish ublox_derive` or `Publish derive`, to make it clearer in the UI which step it is